packages/ra-ui-materialui/src/field/ReferenceOneField.tsx

```diff
@@ -84,7 +87,7 @@ export const ReferenceOneField = <RecordType extends RaRecord = any>(
                 isLoading={isLoading}
                 isFetching={isFetching}
                 referenceRecord={referenceRecord}
-                resourceLinkPath={resourceLinkPath}
+                link={resourceLinkPath}
```

Comment:
If you had to change it here, it will break other components (e.g. Enterprise components), so you should revert this change.